set.mm

```diff
@@ -32795,21 +32801,21 @@ general is seen either by substitution (when the variable ` V ` has no
   $}
 
   ${
-    $d z A $.  $d x z $.
+    $d z A $.  $d z V $.  $d x z $.
     $( Closed theorem form of ~ vtoclgf .  (Contributed by NM, 17-Feb-2013.)
        (Revised by Mario Carneiro, 12-Oct-2016.)  (Proof shortened by JJ,
        11-Aug-2021.)  Avoid ~ ax-13 .  (Revised by Gino Giotto, 6-Oct-2023.) $)
     vtoclgft $p |- ( ( ( F/_ x A /\ F/ x ps )
                   /\ ( A. x ( x = A -> ( ph <-> ps ) )
                      /\ A. x ph ) /\ A e. V ) -> ps ) $=
-      ( vz wnfc wnf wa cv wceq wb wi wal wcel wex wn alnex imbitrid imp elisset
-      w3a nfv nfnfc1 nfcvd nfeqd nfnd nfvd weq eqeq1 notbid a1i 3bitr3g con4bid
-      id cbv2w ad2antrr 3impia biimp imim2i com23 alanimi 19.23t adantl 3adant3
-      mpd ) CDGZBCHZIZCJZDKZABLZMZCNACNIZDEOZUBVKCPZBVIVNVOVPVGVOVPMVHVNVOFJZDK
-      ZFPZVGVPFDEUAVGVSVPVGVRQZFNVKQZCNVSQVPQVGVTWAFCVGFUCCDUDVGVRCVGCVQDVGCVQU
-      EVGUOUFUGVGWAFUHFCUIZVTWALMVGWBVRVKVQVJDUJUKULUPVRFRVKCRUMUNSUQURVIVNVPBM
-      ZVOVIVNWCVNVKBMZCNZVIWCVMAWDCVMAWDVMVKABVLABMVKABUSUTVATVBVHWEWCLVGVKBCVC
-      VDSTVEVF $.
+      ( vz wnfc wnf wa cv wceq wb wi wal wcel wex wn alnex imbitrid imp w3a nfv
+      elissetv nfnfc1 nfcvd id nfeqd nfnd nfvd weq eqeq1 notbid 3bitr3g con4bid
+      a1i cbv2w ad2antrr 3impia biimp imim2i alanimi 19.23t adantl 3adant3 mpd
+      com23 ) CDGZBCHZIZCJZDKZABLZMZCNACNIZDEOZUAVKCPZBVIVNVOVPVGVOVPMVHVNVOFJZ
+      DKZFPZVGVPFDEUCVGVSVPVGVRQZFNVKQZCNVSQVPQVGVTWAFCVGFUBCDUDVGVRCVGCVQDVGCV
+      QUEVGUFUGUHVGWAFUIFCUJZVTWALMVGWBVRVKVQVJDUKULUOUPVRFRVKCRUMUNSUQURVIVNVP
+      BMZVOVIVNWCVNVKBMZCNZVIWCVMAWDCVMAWDVMVKABVLABMVKABUSUTVFTVAVHWEWCLVGVKBC
+      VBVCSTVDVE $.
```

Comment:
What is the advantage of using elissetv instead of elisset? I only see additional DV conditions...

Reply:
The DV conditions are automatically fulfilled with a dummy variable.  I follow an advice given in [elisset](https://us.metamath.org/mpeuni/elisset.html).  And in general I'd say: Use the leanest possible way if alternatives are available.

Reply:
Then  maybe df-clab should be added to " $( $j usage 'vtoclgft' avoids 'ax-13'; $)"

Reply:
Nice observation